prac_08/taxi_simulator.py

```diff
@@ -1,50 +0,0 @@
-"""
-CP1404/CP5632 Practical
-Taxi simulator
-"""
-from prac_08.taxi import Taxi
-from prac_08.silver_service_taxi import SilverServiceTaxi
-
-MENU = """
-Q - Quit
-C - Choose Taxi
-D - Drive"""
-
-
-def main():
-    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
-    total_bill = 0
-    print("Let's drive!")
-    print(MENU)
-
-    menu_choice = str(input(">>> ").lower())
-    while menu_choice != "q":
-        if menu_choice == "c":
-            print("Taxi's Available:")
-            display_taxis(taxis)
-            taxi_choice = int(input("Choose taxi: "))
-            chosen_taxi = taxis[taxi_choice]
```

Comment:
(again haven;t tried running the code) but do the value in the display_taxis function align to the taxis list?... or does the 'for i, taxi in enumerate(taxis):' start at 1 and therefore when retrieving the [1] index from taxis list you end up with the second value instead of taxis[0]?